code/modules/multiz/_HELPERS.dm

```diff
@@ -0,0 +1,138 @@
+/* Multi-Z code was ported from Polaris, which operates under the AGPL v3 license.
+Permission for its use was obtained on 11/12/2017 from Neerti in the Polaris Discord. */
+
+#define isopenspace(A) istype(A, /turf/simulated/open)
+#define OPENSPACE_PLANE_START -23
+#define OPENSPACE_PLANE_END -8
+#define OPENSPACE_PLANE -25
+#define OVER_OPENSPACE_PLANE -7
+//#define QDELETED(X) (!X || X.gc_destroyed)  Polaris has this
+
+/obj/effect/landmark/map_data
+	name = "Unknown"
+	desc = "An unknown location."
+	invisibility = 101
+
+	var/height = 1     ///< The number of Z-Levels in the map.
+	var/turf/edge_type ///< What the map edge should be formed with. (null = world.turf)
+
+/turf/proc/is_space()
+	return 0
+
+/turf/space/is_space()
+	return 1
+
+/atom/proc/set_dir(new_dir) //We should probably use this for our whole codebase for shadows. If z-shadow isn't working I may have to implement it widely.
+	. = new_dir != dir
```

Comment:
This return value is never used as far as I can tell, and we already have `change_dir`. Is this supposed to act differently?